### DISCUSSION TITLE
[fa] Proposed references for Farsi diction

### DISCUSSION THREAD

**Discussion by mokhosh** [Laravel-Lang/common#176] [Ideas]
این دیسکاشن رو به پیشنهاد @ariaieboy عزیز ایجاد کردم که یه سری رفرنس مشخص بکنیم برای رسم‌الخط و انتخاب ترجمه‌های فارسی.
اگر بحث خوبی شکل بگیره و نظرات سنجیده بشه شاید بشه از حد پکیج لاراول فراتر بریم و پیشنهادها رو برای همه نرم‌افزارهای فارسی به اشتراک بذاریم.

توی صحبتی که [اینجا](https://github.com/Laravel-Lang/actions/pull/496) شکل گرفت، این رفرنس‌ها مطرح شد:
- دستور خط فارسی فرهنگستان https://apll.ir/wp-content/uploads/2023/07/Dastour-e-Khat-17.04.1402.pdf
- مدخل شیوه خط فارسی ویکیپدیا https://fa.wikipedia.org/wiki/%D9%88%DB%8C%DA%A9%DB%8C%E2%80%8C%D9%BE%D8%AF%DB%8C%D8%A7:%D8%B4%DB%8C%D9%88%D9%87_%D8%AE%D8%B7_%D9%81%D8%A7%D8%B1%D8%B3%DB%8C

اگر رفرنس دیگه‌ای پیشنهاد میدید یا نظری نسبت به این رفرنس‌ها دارید همینجا قرار بدید تا همفکری کنیم و تصمیم بگیریم.

**Comment by andrey-helldar** [comment]
@ariaieboy, it's for you, I think.

**Comment by ariaieboy** [comment]
فکر میکنم همین رفرنس ها اوکی باشه.
اگر از دوستان دیگه نظری داشت می‌تونه همینجا اعلام کنه تا بررسی بشه
